components/resources/brave_components_strings.grd

```diff
@@ -404,6 +404,7 @@
       <message name="IDS_BRAVE_REWARDS_INTERNALS_CONTRIBUTION_STEP_REWARDS_OFF" desc="">Rewards was turned off</message>
       <message name="IDS_BRAVE_REWARDS_INTERNALS_CONTRIBUTION_STEP_AUTO_CONTRIBUTE_OFF" desc="">Auto-contribute was turned off</message>
       <message name="IDS_BRAVE_REWARDS_INTERNALS_CONTRIBUTION_STEP_RETRY_COUNT" desc="">Stopped retrying</message>
+      <message name="IDS_BRAVE_REWARDS_INTERNALS_MAIN_DISCLAIMER" desc="">All data on this page is sensitive and may include your private keys or wallet IDs. Use abundant caution before sharing.</message>
```

Comment:
maybe rephrase this as:

`These logs may contain sensitive information. Treat them as you would your wallet private keys. Be careful who you share them with.`

Reply:
@diracdeltas we are not talking about just logs here anymore as disclaimer will be shown for all tabs

Reply:
or should we just put disclaimer on log tab?

Reply:
adjusted text. @diracdeltas this is what I have now

![image](https://user-images.githubusercontent.com/9574457/88259772-42d27800-ccc3-11ea-9e94-1bf0abda5ce7.png)